Modules/Microsoft.PowerShell.SecretsManagement/TestExtensionModules/AKVaultScript/SecretManagementExtension/SecretManagementExtension.psm1

```diff
@@ -88,10 +92,22 @@ function Get-SecretInfo
     {
         if ($pattern.IsMatch($vaultSecretInfo.Name))
         {
-            Write-Output ([pscustomobject] @{
-                Name = $vaultSecretInfo.Name
-                Value = "SecureString"
-            })
+            Write-Output (
+                [Microsoft.PowerShell.SecretsManagement.SecretInformation]::new(
```

Comment:
The namespace wasn't updated to `SecretManagement` instead of `SecretsManagement`

Reply:
I didn't change the namespace name because it is the same as the module name, and I felt like it should be consistent.  Do we want to change the module name?